PowerEditor/installer/nsisInclude/tools.nsh

```diff
@@ -172,6 +172,7 @@ Function writeInstallInfoInRegistry
 	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "MajorVersion" "${VERSION_MAJOR}"
 	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "MinorVersion" "${VERSION_MINOR}"
 	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "UninstallString" '"$INSTDIR\uninstall.exe"'
+	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "QuietUninstallString" "$\"$INSTDIR\uninstall.exe$\" /S"
```

Comment:
While your definition is technically correct I would suggest using the same format as with `UninstallString`, which would result in `'"$INSTDIR\uninstall.exe"' /S`.

Only a suggestion to make it more readable and in line with the others.

Reply:
Noticed that, but after all the tests I & the others did and I have the habit of not changing anything after. Guess this time it will be ok, so I am adjusting that.